danieldupriest/reimbursinator
# Added 3rd 'admin' server on port 8445 to serve admin css/js.

## Admin Server CSS

The CSS and JS for the backend administration pages are supposed to be served by a separate server (nginx, apache, etc.). Since I didn't want to mix front-end and back-end web files, I made a new, third "admin" server in docker and served the files from there. It runs on port 8445.

To test:

1. If your docker installation runs on "localhost", you will need to edit `back/reimbursinator/settings.py` and change the `STATIC_URL` line to read `STATIC_URL = 'https://localhost:8445/'`. If you access docker at 192.168.99.100 you can leave it as-is.
2. Start docker. Notice it now loads a third server on startup.
3. Navigate to https://192.168.99.100:8445 (or https://localhost:8445 )
4. Add a certificate exception.
5. Access the backend admin page at https://192.168.99.100:8444/admin/ (or https://localhost:8444/admin/ )
6. Log in with `admin / admin` and enjoy the glorious CSS/Javascriptey goodness!

## Django Version update

In order to fix a django software vulnerability, I updated django to version 2.1.6 in the Pipfile.
